Extension/browser/edge/lib/utils/rules-storage.js

```diff
@@ -15,49 +15,121 @@
  * along with Adguard Browser Extension.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+/* global browser */
+
 /**
  * Filter rules storage implementation.
  *
- * Edge storage has a weird 1MB limit per value. Also Edge does not support "unlimitedStorage" permission.
- * The weird thing is that local storage does not limited like that so we just use it instead.
+ * Unfortunately we have to use this strange logic due to Edge API incompatibilities:
+ * unlimitedStorage permission is supported only from Insider build 15063. Also, before this build Edge storage has a weird 1MB limit per value.
```

Comment:
I am not sure if it is 15063, you can simply say that it is not supported in the Anniversary update version, but it will be supported in the Creators update.